helm/panda/charts/server/values.yaml

```diff
@@ -72,8 +72,8 @@ securityContext: {}
 
 service:
   type: ClusterIP
-  httpPort: 80
-  httpsPort: 443
+  httpPort: 25080
```

Comment:
This change is not good for OKD since Route cannot forward access to non-standard ports. Why do you need this change?

Reply:
I misunderstood the k8s service. I thought the service port should be the same as the deployment port. I just checked the documents and found it can be different. I will roll it back.